graylog2-web-interface/src/hooks/useCurrentUser.ts

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (C) 2020 Graylog, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the Server Side Public License, version 1,
+ * as published by MongoDB, Inc.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * Server Side Public License for more details.
+ *
+ * You should have received a copy of the Server Side Public License
+ * along with this program. If not, see
+ * <http://www.mongodb.com/licensing/server-side-public-license>.
+ */
+import { useContext } from 'react';
+
+import CurrentUserContext from 'contexts/CurrentUserContext';
```

Comment:
At some point, I think it would be a good idea to get rid of the `CurrentUserContext` in favor of the hook.

Reply:
I have not thought about it, but it is true at some point we no longer need the context.